frontend/src/pages/AccountSend.vue

```diff
@@ -1092,7 +1085,7 @@ function useSendForm() {
         if (token.symbol !== NATIVE_TOKEN.value.symbol) {
           const tokenContract = new Contract(token.address, ERC20_ABI, signer.value);
           const batchSendAddress = umbra.value?.batchSendContract!.address;
-          const parsedAmount = parseUnits(amount.replace(/,/g, '') || '0', token.decimals);
+          const parsedAmount = parseUnits(amount.toString() || '0', token.decimals);
```

Comment:
Amount is a `BigNumber` so I don't think we'll ever hit the `|| '0'` branch right?
```suggestion
          const parsedAmount = parseUnits(amount.toString(), token.decimals);
```

Also I'm a bit confused about what's going on here and not sure if this is correct. Above, we set values in the map using `const updatedAmount = previousAmount.add(parseUnits(send.amount, send.token.decimals))` which means that amount should already have the correct decimals in the map. For example, 1 USDC gets saved here as 1000000, which means that on this line we are again scaling up that amount by 6 decimals, which seems incorrect. Am I misunderstanding something here?